src/components/advanced-color-picker/advanced-color-picker.component.js

```diff
@@ -50,8 +50,20 @@ const AdvancedColorPicker = ({
     }
   }, [colors, currentColor, dialogOpen, isOpen]);
 
-  const handleFocus = useCallback(() => {
-    selectedColorRef.current.input.current.focus();
+  const handleFocus = useCallback((e, firstFocusableElement, lastFocusableElement) => {
+    if (e.key === 'Tab' || e.keyCode === 9) {
```

Comment:
`keyCode` is deprecated, isn't `e.key === 'Tab'` enough?
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
It would be yes, I didn't realise `keyCode` was deprecated - just copied the existing logic in the component, will change that